src/app/cluster/details/shared/version-change-dialog/style.scss

```diff
@@ -12,6 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+@use 'variables';
+
 .checkbox-wrapper {
   min-height: 35px;
 }
+
+.dynamic-kubelet-config-nodes {
+  font-size: variables.$font-size-body;
```

Comment:
You don't need to apply font size there as it is in the parent class. Also, check if we have some global class for making the font weight bigger (bolder).